src/mlpack/methods/reinforcement_learning/replay/random_replay.hpp

```diff
@@ -0,0 +1,142 @@
+/**
+ * @file random_replay.hpp
```

Comment:
> Yeah there is also prioritized replay but I'm not going to implement it in the near future. But I sill think a separate folder is better because once we add a new replay method, a separate folder can provide backward compatibility automatically. Otherwise user may have to modify their `#include` clause.

Hi, @ShangtongZhang, do you have any plan to implement prioritized replay buffer ? I would like to contribute as a process to be familiar with the mlpack codebase and get involved in this community. I also  implement it in Python before. [PER Implementation](https://github.com/robotcator/Assignment)

Reply:
@robotcator Yeah, it would be great if you can contribute PER.

Reply:
Sorry for late reply, I miss the notification. I written the schedule for PER and I think I will begin to code soon.